### PR TITLE
Mobile_ Porfolio_Setup: Desktop Version

### PR DESCRIPTION
**Pull request by 3nityGitHub**
## In this PR:
- I included media queries to the pages can be compatible with large screens like desktops
- I modified index.HTML files to suit the media query
- I modified styles.CSS to suit the media query

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -63,7 +64,7 @@ <h3 class="multi_story">Multi-Post Stories</h3>
   </div>
 
   <div class="recent_work_container flex">
- <div class="first_project">
+    <div class="first_project">
     <h3 class="first_project_title">Profesional Art Printing Data</h3>
     <p class="first_project_content">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
     <ul class="first_project_skill_list flex">
```
Comment (roshan-bajgain) [inline index.html:70]: - [ ] I would suggest you please stick to the design as much as possible. your logo and content should be inline on the same line and also increase the width of the `p` tag so it matches the design. See the image below for reference. The text is aligned to the left but it doesn't pass the middle of the page.
### Your design
his

### Figma design
fimga

File: index.html [hunk 2]
```diff
@@ -20,6 +20,7 @@ <h1 class="nav_brand">Rapheal Okonta</h1>
         <li class="nav_link"><a href="#">Portfolio</a></li>
         <li class="nav_link"><a href="#">About</a></li>
         <li class="nav_link"><a href="#">Contact</a></li>
+        <li class="nav_link_email"><a href="#"><img src="image folder/Disabled.png" alt=""></a></li>
```
Comment (roshan-bajgain) [inline index.html:23]: - [ ] I would suggest you please stick to the design as much as possible. your logo and content should be inline on the same line and also increase the width of the `p` tag so it matches the design. See the image below for reference. The text is aligned to the left but it doesn't pass the middle of the page.
### Your design
his

### Figma design
fimga

File: index.html [hunk 3]
```diff
@@ -63,7 +64,7 @@ <h3 class="multi_story">Multi-Post Stories</h3>
   </div>
 
   <div class="recent_work_container flex">
- <div class="first_project">
+    <div class="first_project">
```
Comment (roshan-bajgain) [inline index.html:67]: - [ ]  i would suggest you please stick to the design as much as possible. The featured project image takes 60% of the page's width and is aligned to the left, the description takes the remaining 40% and is aligned to the right. There is equal padding from left and right. Please see the image below for reference.
- [ ] Kindly increase the font size of the placeholder of the `see-project` buttons.
### Your design
his
### Figma design

figma

File: index.html [hunk 4]
```diff
@@ -83,9 +84,9 @@ <h3 class="second_project_title">Profesional Art Printing Data</h3>
...
     </ul>
     <button class="see_project_works2">See Project</button>
-  </div>
+   </div>
 
-  <div class="third_project">
+   <div class="third_project">
     <h3 class="third_project_title">Profesional Art Printing Data</h3>
     <p class="third_project_content">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
     <ul class="third_project_skill_list flex">
```
Comment (roshan-bajgain) [inline index.html:92]: - [ ] I would suggest please adjust the spacing on the left and right of the page. Also, stick to the design as much as possible. please fix the background and color of the button.
- [ ] please implement in all the cards
### Your design
yourrrrr

### Figma design
fogma

**Review by roshan-bajgain — CHANGES_REQUESTED** [review]
Hi @3nityGitHub, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️
## Required Changes ♻️
- [ ] Your mobile design does not look as expected. please fix the horizontal scrolling. It has a problem with horizontal scrolling. please fix it
### Design
Screenshot 2022-12-09 143217

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Debas-31 — CHANGES_REQUESTED** [review]
> ## Hi @3nityGitHub :wave:, 
# Status: Require Change :recycle:
Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.




 ## Highlights
- No linters error  :heavy_check_mark:
- PR title and summary :heavy_check_mark: 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by abdulhamiid — APPROVED** [review]
# :green_circle:  Approved! :partying_face:
Hi @3nityGitHub , 
image
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 🌟

- ✔️ Descriptions are very descriptive
- ✔️ Linters checks are passing
- ✔️ Use GitHub Flow
- ✔️ Meaningful commit messages.

## Optional suggestions 🤷‍♂️
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

## Important Notes 📝
- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**